### DISCUSSION TITLE
no function or associated item named `new` found for struct `Runtime` in the current scope

### DISCUSSION THREAD

**Discussion by 2-3-5-41** [tokio-rs/tokio#5918] [Q&A] [answered]
Idk if this is a bug, or improper documentation that led me here; but using this snippet from the [Tokio Docs](https://docs.rs/tokio/latest/tokio/runtime/struct.Runtime.html#method.new), I'm left with this error: ```no function or associated item named `new` found for struct `Runtime` in the current scope```.

For more context, this is my code:
```rs
use std::sync::Arc;
use tokio::runtime::Runtime;

pub struct TokioRuntime {
    pub runtime: Arc<Runtime>
}

impl TokioRuntime {
    fn init() -> Self {
        let rt = Runtime::new(); // <--- This line is pushing this error to the compiler.
        Self { runtime: Arc::new(rt) }
    }
}
```

**Comment by sfackler** [comment, marked as answer]
Did you enable the `rt-multi-thread` Cargo feature?

**Reply by 2-3-5-41** [reply]
That does resolve the error, I heavily overlooked several parts in the documentation.
